src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carRentOfDay = 40;
+  const longTermRent = 7;
+  const shortTermRent = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+  const sumRentCar = carRentOfDay * days;
+
+  if (days >= longTermRent) {
+    return sumRentCar - longTermDiscount;
```

Comment:
The use of `if` with `return` here is good as it simplifies the logic and avoids unnecessary `else` statements.